### PR TITLE
PlayerPed and "Redefine controls" menu done, fixes

### PR DESCRIPTION
**Pull request by erorcun**

### CODE REVIEW

**Review by ShFil119 — APPROVED** [review]

File: src/core/Stats.cpp [hunk 1]
```diff
@@ -773,4 +776,255 @@ CStats::PopulateFavoriteRadioStationList()
...
+
+	STAT_LINE("PER_COM", &percentCompleted, false, nil);
+	STAT_LINE("NMISON", &MissionsGiven, false, nil);
+	STAT_LINE("FEST_MP", &MissionsPassed, false, &TotalNumberMissions);
+	if (CGame::nastyGame) {
+		STAT_LINE("FEST_RP", &NumberKillFrenziesPassed, false, &TotalNumberKillFrenzies);
+	}
+	
+	CPlayerInfo &player = CWorld::Players[CWorld::PlayerInFocus];
+	float packagesPercent = 0.0f;
```
Comment (Nick007J) [inline src/core/Stats.cpp:903]: this is just asking for FIX_BUGS :D

File: src/peds/PlayerPed.cpp [hunk 2]
```diff
@@ -1814,6 +2052,79 @@ CPlayerPed::PlayIdleAnimations(CPad *padUsed)
...
+						m_pMeleeList[i] = nil;
+				} else {
+					m_pMeleeList[i] = nil;
+				}
+			} else {
+				m_pMeleeList[i] = nil;
+			}
+		}
+	}
+	m_bNoOneLeftInMeleeList = m_pMeleeList[0] == this && m_pMeleeList[1] == this && m_pMeleeList[2] == this && m_pMeleeList[4] == this && m_pMeleeList[5] == this;
```
Comment (Nick007J) [inline src/peds/PlayerPed.cpp:2125]: can it be an unrolled loop?

**Review by Nick007J — APPROVED** [review]
too dumb to understand frontend, rest looks good

File: src/peds/PlayerPed.cpp (hunk 2 above)
Comment (erorcun) [reply src/peds/PlayerPed.cpp]: oh haha you made me realize they missed `m_pMeleeList[3] == this`, so it wasn't a loop :p also while looking to it I understood the melee list better, so I also pushed some renamings/comments along with the fix.

File: src/core/Stats.cpp (hunk 1 above)
Comment (erorcun) [reply src/core/Stats.cpp:903]: hehe ok, will fix it in next PR

**Pull request merged**